project/plugins.sbt

```diff
@@ -6,7 +6,7 @@ resolvers += Resolver.url(
   Resolver.ivyStylePatterns
 )
 
-addSbtPlugin("uk.gov.hmrc"      % "sbt-auto-build"        % "3.20.0")
+addSbtPlugin("uk.gov.hmrc"      % "sbt-auto-build"        % "3.21.0")
 addSbtPlugin("org.scalameta"    % "sbt-scalafmt"          % "2.5.2")
 addSbtPlugin("com.timushev.sbt" % "sbt-updates"           % "0.6.4")
 addSbtPlugin("org.scalastyle"   % "scalastyle-sbt-plugin" % "1.0.0" exclude ("org.scala-lang.modules", "scala-xml_2.12"))
```

Comment:
Please update [sbt-test-report](https://github.com/hmrc/sbt-test-report), [ui-test-runner](https://github.com/hmrc/ui-test-runner), [cucumber-jvm-scala](https://github.com/cucumber/cucumber-jvm-scala) and [cucumber-jvm](https://github.com/cucumber/cucumber-jvm), thanks